### PR TITLE
feat(upgradelog): add fleet-controller and local agent's log

### PR DESCRIPTION
**Pull request by starbops**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**

We want to examine Fleet relevant logs output during upgrades.

**Solution:**

Add `cattle-fleet-system/fleet-controller` and `cattle-fleet-local-system/fleet-agent` into ClusterFlow's match filter

**Related Issue:**

#5335 

**Test plan:**

1. Prepare a Harvester cluster with a custom build containing the fix
2. Upgrade the cluster using the same ISO image
3. Download the upgrade log archive from the dashboard
4. Check if the downloaded and extracted archive contains fleet-related logs

### CODE REVIEW

**Review by bk201 — APPROVED** [review]

File: pkg/controller/master/upgradelog/common.go [hunk 1]
```diff
@@ -238,6 +238,22 @@ func prepareClusterFlow(upgradeLog *harvesterv1.UpgradeLog) *loggingv1.ClusterFl
 						ContainerNames: []string{"upgrade"},
 					},
 				},
+				{
+					ClusterSelect: &loggingv1.ClusterSelect{
+						Labels: map[string]string{
+							"app": "fleet-controller",
+						},
+						Namespaces: []string{"cattle-fleet-agent"},
```
Comment (w13915984028) [inline pkg/controller/master/upgradelog/common.go]: Are both namespaces right?

`cattle-fleet-system/fleet-controller and cattle-fleet-local-system/fleet-agent`

Comment (starbops) [reply pkg/controller/master/upgradelog/common.go]: Updated, thanks!

**Review by w13915984028 — APPROVED** [review]
LGTM, thanks.

**Comment by bk201** [comment]
@mergifyio backport v1.3

**Pull request merged**